1031/en.md

```diff
@@ -0,0 +1,77 @@
+# **LOJ 1031 - Easy Game**
+
+## **Solution Idea**
+You can solve the problem using Dynamic Programming concept.
+
+- Create a prefix_sum array. 
+- Think about the DP state. When you are in range i to j what could be the answer ? 
+- you should store maximum difference in dp.
+- When you want to compute ans for range i to j : 
+    - There can be 3 possible ans:
+        - You can take whole range i to j
+        - You can take one or more numbers from the left-end
+        - You can take one or more numbers from the right-end
+    - So the idea will be as follow
+        - `dp[i][j] = prefix_sum(taken_range) - dp of other range`
+- for base case : Think about what will be the ans when n = 1 ? 
```

Comment:
Feels like too much implementation focused details. Let's try to rephrase this section. Let's state the _states_, _transitions_ and _base cases_ a bit more clearly and without using code.